src/foam/u2/search/Toolbar.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.search',
+  name: 'Toolbar',
+  extends: 'foam.u2.View',
+  properties: [
+    {
+      class: 'FObjectProperty',
+      of: 'foam.mlang.predicate.Predicate',
+      name: 'data',
+      factory: function() {
+        return foam.mlang.predicate.True.create();
+      }
+    },
+    {
+      class: 'String', // TODO remove.
+      name: 'data_',
```

Comment:
Maybe just call this "search" instead of data_ ?

Reply:
Done.